src/tests/system/tests/test_ipa_trusts.py

```diff
@@ -15,7 +15,7 @@
 @pytest.mark.importance("low")
 @pytest.mark.ticket(jira="RHEL-3925", gh=6942)
 @pytest.mark.topology(KnownTopologyGroup.IPATrust)
-def test_trusts__lookup_group_without_sid(ipa: IPA, trusted: GenericADProvider):
+def test_ipa_trusts__lookup_group_without_sid(ipa: IPA, trusted: GenericADProvider):
     """
     :title: Subdomain goes offline if IPA group is missing SID
```

Comment:
In my opinion the title 'Subdomain stays online if IPA group is missing SID' is a more accurate description of this test.